c++/src/capnp/capability.h

```diff
@@ -656,6 +686,9 @@ class RequestHook {
   virtual kj::Promise<void> sendStreaming() = 0;
   // Send a streaming call.
 
+  virtual AnyPointer::Pipeline sendForPipeline() = 0;
```

Comment:
Should this have had a default implementation? Otherwise, this is a source-incompatible change.

Reply:
I kind of wanted anyone implementing these interfaces -- which is not many -- to have to think about how to implement this, rather than just get an inefficient default implementation without ever realizing it.

I forgot to check if the Workers codebase included any implementers, though. Sorry about that.